MSAL/src/requests/MSALTokenResponse.m

```diff
@@ -26,7 +26,7 @@
 //------------------------------------------------------------------------------
 
 #import "MSALTokenResponse.h"
-#import "MSALClientInfo.h"
+#import "MSIDClientInfo.h"
```

Comment:
do we still need MSALTokenResponse?

Reply:
removed

Reply:
Hmmm, it doesn't show the removal change

Reply:
right, removed it's usage from the code, but didn't delete the files. Fixed.